.env

```diff
@@ -1,2 +1,3 @@
 PORT=9000
-NODE_ENV=dev
\ No newline at end of file
+NODE_ENV=dev
+MONGO_URL=mongodb://localhost:27017/express-training
```

Comment:
This file needs to be added to gitignore.

Reply:
.env file added to gitignore